src/redis-cli.c

```diff
@@ -1194,7 +1197,11 @@ static sds cliFormatReplyJson(sds out, redisReply *r) {
         break;
     case REDIS_REPLY_STRING:
     case REDIS_REPLY_VERB:
-        out = sdscatrepr(out,r->str,r->len);
+        if (binary == 1) {
```

Comment:
Why use `== 1` either match it against JSON_ENABLE_BINARY_ENCODING, or just use it as a boolean.